src/app/Task1/task1.js

```diff
@@ -0,0 +1,5 @@
+var s = "";
+for (var i = 0; i < 7; i++) {
+  s += "#";
+  console.log(s);
+}
```

Comment:
It is the simplest way indeed. However you can use not only integer counter, but use any other criteria in for-cycle. Basically, you can get rid of declaring s variable out of cycle and in the cycle body, only describing it the loop. Try out smth like 
```
for (var s= "#"; s.length() <= 7; s+="#") {...}
```